tran_kien_giang/extend.php

```diff
@@ -0,0 +1,46 @@
+<?php 
+	$so_tien_can_quy_doi=$_GET['so_tien_can_quy_doi'];
+	$option=$_GET['option'];
+	if ($option == 'VND -> USD') {
+	    $so_tien_da_quy_doi=$so_tien_can_quy_doi * 0.000044;
+	    $format_number_input = number_format($so_tien_can_quy_doi, 0, '', ',') .' VND';
+	    $format_number_output = number_format($so_tien_da_quy_doi, 3, '.', '') .' USD';
+	}
+	else if ($option == 'USD -> VND') {
```

Comment:
```suggestion
	} elseif ($option == 'USD -> VND') {
```
elseif viết như này nhé